### ISSUE TITLE
Pending tabs all start loading when I drop them in another window

### ISSUE THREAD

**Issue by arni2033** [piroor/multipletab#132]
My Info:   Win7_64, Nightly 49, 32bit, ID 20160526082509 (2016-05-26)

STR_1:
1. Open new window (with about:home), open https://www.youtube.com/watch?v=mD3v1B_aXw0 in a new tab , open new tab (Ctrl+T), load about:home instead of about:newtab
2. Close and restore that window
3. Click on the 1st tab, hold Shift, click on the last tab (to select 3 tabs)
4. Hover mouse over the 1st tab, hold left mouse button, drag and drop tabs to a window existed before Step 1

AR:
 Pending tab with youtube video starts loading, unwanted sound appears.
 In real life, browser just starts loading a lot of pages, including youtube videos
ER:
 Pending tabs shouldn't load unless user selected them specifically

Workaround:
 Disable internet connection before drag-n-drop operation, and wait until all dropped tabs will show "Connection error" page. So far this is the only way to avoid unwanted traffic/sound/RAM consumption.

**Comment by piroor** [comment]
Moving pending tabs across windows breaks those tabs. To avoid the problem I decided to restore selected tabs before moved, because it is easiest and safe method to do it.

On the other hand, moving pending tabs as pending can be done if I deeply hack session management system of Firefox itself. But currently I don't have enough time to implement (and maintain) it. Moreover, if I did it, MTH will become more hard to update for Firefox's future versions...
